### DISCUSSION TITLE
CS8063 - Improve wording

### DISCUSSION THREAD

**Discussion by The-Futurist** [dotnet/roslyn#67523] [General]
This has been bothering me for some time:

![image](https://user-images.githubusercontent.com/12262952/228328821-888857f8-21c4-4128-8a2e-c36ff8557a74.png)

The message reads `'text' may be null here.`

This is not really good English because 'may" means optional, permissible, something one could choose to allow, rather than undesirable, to be avoided, so this should really be changed to:  `'text' might be null here.`

This might be particularly confusing to non native English speakers too.

**Comment by CyrusNajmabadi** [comment]
Moved to https://github.com/dotnet/roslyn/issues/67525.
